tests/components/suez_water/test_config_flow.py

```diff
@@ -15,20 +15,21 @@
 from tests.common import MockConfigEntry
 
 
-async def test_form(hass: HomeAssistant, mock_setup_entry: AsyncMock) -> None:
+async def test_form(
+    hass: HomeAssistant, mock_setup_entry: AsyncMock, suez_client
```

Comment:
```suggestion
    hass: HomeAssistant, mock_setup_entry: AsyncMock, suez_client: AsyncMock
```
Please do this for the others as well

Reply:
done for all tests